src/views/blocks/rsvp/content-inactive.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Block: RSVP
+ * Inactive Content
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/blocks/rsvp/content-inactive.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version 4.9
+ *
+ */
+
+$message = $this->get( 'active_past' ) ? esc_html__( 'RSVPs are no longer available', 'event-tickets' ) : esc_html__( 'RSVPs are not yet available', 'event-tickets' );
```

Comment:
What is this variable? Should we add some comments with context? or rename it?

Reply:
This is to check if all RSVPs are not available and are in the past. In the case there are multiple RSVPs and one RSVP is in the future, the second message will show.